linebot/response.go

```diff
@@ -76,6 +76,68 @@ type MessageConsumptionResponse struct {
 	TotalUsage int64
 }
 
+// MessagesNumberDeliveryResponse type
+type MessagesNumberDeliveryResponse struct {
+	Status          string `json:"status"`
+	Broadcast       int64  `json:"broadcast"`
+	Targeting       int64  `json:"targeting"`
+	AutoResponse    int64  `json:"autoResponse"`
+	WelcomeResponse int64  `json:"welcomeResponse"`
+	Chat            int64  `json:"chat"`
+	APIBroadcast    int64  `json:"apiBroadcast"`
+	APIPush         int64  `json:"apiPush"`
+	APIMulticast    int64  `json:"apiMulticast"`
+	APIReply        int64  `json:"apiReply"`
+}
+
+// MessagesNumberFollowersResponse type
+type MessagesNumberFollowersResponse struct {
+	Status          string `json:"status"`
+	Followers       int64  `json:"followers"`
+	TargetedReaches int64  `json:"targetedReaches"`
+	Blocks          int64  `json:"blocks"`
+}
+
+// MessagesFriendDemographicsResponse type
+type MessagesFriendDemographicsResponse struct {
+	Available           bool                       `json:"available"`
+	Genders             []GenderDetail             `json:"genders"`
+	Ages                []AgeDetail                `json:"ages"`
+	Areas               []AreaArrayDetail          `json:"areas"`
+	AppTypes            []AppTypeDetail            `json:"appTypes"`
+	SubscriptionPeriods []SubscriptionPeriodDetail `json:"subscriptionPeriods"`
+}
+
+// GenderDetail type
+type GenderDetail struct {
+	Gender     string  `json:"gender"`
+	Percentage float64 `json:"percentage"`
+}
+
+// AgeDetail type
+type AgeDetail struct {
+	Age        string  `json:"Age"`
```

Comment:
I think this JSON field name should be lowercased as `age`, I suppose. `TestGetFriendDemographics` might have room to improve by adding the `age` field in its JSON structure.

Reply:
Fixed, and `TestGetFriendDemographics` already have `age` field. Thank you.